nodes/scheduling/nodes-custom-scheduler.adoc

```diff
@@ -0,0 +1,25 @@
+[id="nodes-custom-scheduler"]
+= Running a custom scheduler
+include::modules/common-attributes.adoc[]
+:context: nodes-custom-scheduler
+
+toc::[]
+
+You can run multiple custom schedulers alongside the default scheduler and configure which scheduler to use for each Pod.
+
+To schedule a given Pod using a specific scheduler, xref:../../nodes/scheduling/nodes-custom-scheduler.adoc#nodes-custom-scheduler-deploying_nodes-custom-scheduler[specify the name of the scheduler in that Pod specification].
+
+[NOTE]
+====
+Each scheduler should operate over its own set of nodes. If two or more schedulers operate on the same node, they might schedule more Pods on the node than there are available resources for. Pods might get rejected due to insufficient resources in this case.
```

Comment:
@ingvagabund Can you please check the wording of this?

Reply:
Each scheduler has a separate view of resources in a cluster. For that reason, every scheduler should operate over its own set of nodes.

If two or more schedulers operate on the same node, they might intervene with each other and schedule more Pods on the same node ...